### ISSUE TITLE
Allow for >1 Random Basic Weakness

### ISSUE THREAD

**Issue by mattsigal** [Kamalisk/arkhamdb#31]
I love the new default for the "Random Weakness" placeholder when building decks. That is wonderful!  However, I would like to suggest that we allow this to have a new maximum of 5 copies of "Random Basic Weakness" though. If we are building for a standalone adventure, we can incorporate:

![screen shot 2016-12-08 at 11 31 04](https://cloud.githubusercontent.com/assets/1421758/21018160/d39d4eca-bd39-11e6-9033-fed9856ca949.png)

Obviously most people won't max that out (gg to them if they do), but it would be nice to give that flexibility.

Right now I'm building for a Rougarou session and am planning on using 19XP (so 2 basic weaknesses). If I was to publish the deck, it would be great to show that.

**Comment by sacrelicious2** [comment]
One thing you can do is add a random basic weakness, click the button to have it select a weakness, then add random basic weakness again and the click it again.

**Comment by Kamalisk** [comment]
it should be possible to set the limit to 5 for that special card, just depends how it looks with it being so wide.

**Comment by mattsigal** [comment]
Yeah, it will be a bit wide. Maybe it could be a dropdown menu instead of side by side? Or at least if you search for "Random Basic Weakness", you could enter >2?

An aside, that particular search has no associated image:

![screen shot 2016-12-10 at 10 54 31](https://cloud.githubusercontent.com/assets/1421758/21074436/279a8b1c-bec7-11e6-90a0-81176114c895.png)
